ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -37,6 +37,12 @@ FAN:
   implements:
   - EQUIPMENT
 
+DSU:
```

Comment:
Is this a PCU? If not, how is it distinguished from a PCU? This name is not defined on the BDNS or BOS Abbreviation standard.

Reply:
It is a type of a PCU (pollution control Unit) which is takes of the particulate matters form the exhaust air before it is left outside. Since it is a dry scrubber, we have requested for inclusion of the name under BDNS pull request 112.